level.py

```diff
@@ -40,3 +42,38 @@ def generate_grid_tile(self, x, y):
             return TileType.BRICK
         else:
             return TileType.DIRT
+
+
+class SaveLevel(Level):
+    def __init__(self,  path):
+        tile_map = tmx.TileMap.load(path)
+        layer = tile_map.layers[0]
+        self.width = tile_map.width
+        self.height = tile_map.height
+        self.grid = [
+            [None for x in range(self.width)] for y in range(self.height)
+        ]
+        for index, tile in enumerate(layer.tiles):
+            x = index % self.width
+            y = index // self.width
+            if tile.gid == 54: # bush block
+                self.grid[y][x] = TileType.BUSH
```

Comment:
We're extending from Level here which has a "getTile" method which would probably be better to use.